myuw/templates/handlebars/teaching/photo_list.html

```diff
@@ -2,103 +2,216 @@
     {% tplhandlebars "photo_class_list" %}
 
 <div class="container">
-    <div class="row">
-        <div id="photoclasslist_popup">
+
+        <div id="photoclasslist_popup" class="myuw-photo-list">
+
             <div class="card"  data-name="PhotoClasslistPopupCard" data-type="card">
+
                 <h3 class="scheduleHeading">{{ toTitleCase quarter }} {{ year }} | {{ sections.0.curriculum_abbr }} {{ sections.0.course_number }} {{ sections.0.section_id }} | SLN {{ sections.0.sln }}</h3>
-                <div id="classlist_controls" class="clearfix" style="padding-bottom: 40px;">
-                    <div class="classlist-styled-select pull-left">Sort by:
-                        <select id="sort_list">
-                            <option value="name,surname">First Name</option>
-                            <option value="surname,name" selected="selected">Last Name</option>
-                            <option value="netid">UW NetID</option>
-                            <option value="class_code,surname,name">Class</option>
-                            <option value="credits,surname,name">Credits</option>
-                            {{ #each sections.0.linked_types }}
-                            <option value="linked_type_{{ this }},surname,name">{{ ucfirst this }} Section</option>
-                            {{ /each }}
-                        </select>
 
+                <div id="classlist_controls" class="row myuw-photo-list-controls" style="padding-bottom: 40px;">
+
+                    <div class="col-sm-5">
+                        Display:&nbsp;&nbsp;<a id="list_view" href="#"><i class="fa fa-list" aria-hidden="true"></i> Detailed (List)</a> | <a id="grid_view" href="#"><i class="fa fa-th" aria-hidden="true"></i> Simple (Grid)</a>
                     </div>
-                    <div class="pull-right">
-                        <a href="#" id="download_class_list"><i class="fa fa-download"></i>&nbsp;Download class list (Excel)</a> | <a href="javascript:window.print()" class="classlist-print"><i class="fa fa-print"></i>&nbsp;print</a>
+                    <div class="col-sm-3">
```

Comment:
![myuw__photo_class_list](https://user-images.githubusercontent.com/6953502/27353236-35a63e1c-55b8-11e7-8bb6-2eeb9bfb1c55.png)

Wondering if we put these controls in the same col-sm-x div if they would align better? Maybe there's another way.